packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/property-table/use-get-cell-content.ts

```diff
@@ -0,0 +1,81 @@
+import { GridCell, GridCellKind, Item } from "@glideapps/glide-data-grid";
+import { useCallback } from "react";
+import { useEntityEditor } from "../entity-editor-context";
+import { propertyGridIndexes } from "./constants";
+import { PropertyRow } from "./types";
+
+export const useGetCellContent = (rowData: PropertyRow[]) => {
+  const { propertySort } = useEntityEditor();
+
+  const getCellContent = useCallback(
+    ([col, row]: Item): GridCell => {
+      const property = rowData[row];
+
+      if (!property) {
+        throw new Error();
+      }
+
+      const propertyKey = propertyGridIndexes[col];
+
+      if (!propertyKey) {
+        throw new Error();
```

Comment:
Do these need descriptions?

Reply:
If this error pops up, it means we're using glide grid horribly wrong. Added this to satisfy typescript for the rest of the scope